drivers/claims_reporting/app/controllers/claims_reporting/warehouse_reports/reconciliation_controller.rb

```diff
@@ -11,6 +11,12 @@ class ReconciliationController < ApplicationController
     def index
       @report = ClaimsReporting::ReconcilationReport.new(**filter_params)
       @file = ::ClaimsReporting::CpPaymentUpload.new
+      respond_to do |format|
+        format.html {} # render the default template
+        format.csv do
+          send_data @report.to_csv, filename: "unmatched-claims-#{Time.now}.csv"
```

Comment:
I believe you want `Time.current.to_s(:db)` because Time.now can have commas which messes with Windows.

Reply:
Got it, i had copied code i found elsewhere in openpath, do you want me to change those?

Reply:
Sure!  We've been slowly updating when we find them.

Reply:
Did a search and replace but used `to_s(:number)` which is the all number version of `:db` to be extra safe

Reply:
That's not as readable, but for Time.current, I guess it makes more sense since it includes colons, which macs don't like.